apps/api/src/applicant/external-api.service.ts

```diff
@@ -650,13 +649,28 @@ export class ExternalAPIService {
     }
   }
 
-  async getApplicants(start?: string, end?: string) {
+  async getApplicants(filter: IENUserFilterAPIDTO) {
+    const { from, to, limit, skip } = filter;
+    let queryString = `SELECT DISTINCT applicant_id  
+    FROM ien_applicant_status_audit 
+    WHERE updated_date <'${to || dayjs().add(1, 'day').format('YYYY-MM-DD')}' and updated_date >'${
+      from || '1914-07-18'
+    }'`;
+    if (limit) {
+      queryString = queryString + (limit ? ` LIMIT ${limit} ` : '');
```

Comment:
with `if (limit)`, `limit ?` seems redundant.

Reply:
You're right 😂 . When I wrote the line first, I had it as part of the declaration for QueryString, so I had to make sure it was null safe. I'll fix this.